src/harness/fourslashImpl.ts

```diff
@@ -2449,7 +2449,50 @@ namespace FourSlash {
             Harness.IO.log(this.spanInfoToString(this.getNameOrDottedNameSpan(pos)!, "**"));
         }
 
-        private verifyClassifications(expected: { classificationType: string; text: string; textSpan?: TextSpan }[], actual: ts.ClassifiedSpan[], sourceFileText: string) {
+        private classificationToIdentifier(classification: number){
+
+            const tokenTypes: string[] = [];
+            tokenTypes[ts.classifier.vscode.TokenType.class] = "class";
+            tokenTypes[ts.classifier.vscode.TokenType.enum] = "enum";
+            tokenTypes[ts.classifier.vscode.TokenType.interface] = "interface";
+            tokenTypes[ts.classifier.vscode.TokenType.namespace] = "namespace";
+            tokenTypes[ts.classifier.vscode.TokenType.typeParameter] = "typeParameter";
+            tokenTypes[ts.classifier.vscode.TokenType.type] = "type";
+            tokenTypes[ts.classifier.vscode.TokenType.parameter] = "parameter";
+            tokenTypes[ts.classifier.vscode.TokenType.variable] = "variable";
+            tokenTypes[ts.classifier.vscode.TokenType.enumMember] = "enumMember";
+            tokenTypes[ts.classifier.vscode.TokenType.property] = "property";
+            tokenTypes[ts.classifier.vscode.TokenType.function] = "function";
+            tokenTypes[ts.classifier.vscode.TokenType.member] = "member";
+
+            const tokenModifiers: string[] = [];
+            tokenModifiers[ts.classifier.vscode.TokenModifier.async] = "async";
+            tokenModifiers[ts.classifier.vscode.TokenModifier.declaration] = "declaration";
+            tokenModifiers[ts.classifier.vscode.TokenModifier.readonly] = "readonly";
+            tokenModifiers[ts.classifier.vscode.TokenModifier.static] = "static";
+            tokenModifiers[ts.classifier.vscode.TokenModifier.local] = "local";
+            tokenModifiers[ts.classifier.vscode.TokenModifier.defaultLibrary] = "defaultLibrary";
+
+
+            function getTokenTypeFromClassification(tsClassification: number): number | undefined {
+                if (tsClassification > ts.classifier.vscode.TokenEncodingConsts.modifierMask) {
+                    return (tsClassification >> ts.classifier.vscode.TokenEncodingConsts.typeOffset) - 1;
+                }
+                return undefined;
+            }
+
+            function getTokenModifierFromClassification(tsClassification: number) {
+                return tsClassification & ts.classifier.vscode.TokenEncodingConsts.modifierMask;
+            }
+
+            const typeIdx = getTokenTypeFromClassification(classification) || 0;
+            const modSet = getTokenModifierFromClassification(classification);
+
+            const tokenClassifiction = [tokenTypes[typeIdx], ...tokenModifiers.filter((_, i) => modSet & 1 << i)].join(".");
+            return tokenClassifiction;

```

Comment:
typo: tokenClassifiction, but I'd recommend inlining entirely

```suggestion
            return [tokenTypes[typeIdx], ...tokenModifiers.filter((_, i) => modSet & 1 << i)].join(".");
```